### PR TITLE
Add logger support

### PR DESCRIPTION
**Pull request by chiastolite**
example
---

config.middleware.use Rack::Lineprof, logger: Logger.new('./log/Lineprof.log')

### CODE REVIEW

File: lib/rack/lineprof.rb [hunk 1]
```diff
@@ -21,13 +22,14 @@ def initialize app, options = {}
...
       matcher = request.params['lineprof'] || options[:profile]
+      logger  = options[:logger] || ::Logger.new(STDOUT)
 
       return @app.call env unless matcher
 
       response = nil
       profile = lineprof(%r{#{matcher}}) { response = @app.call env }
 
-      puts Term::ANSIColor.blue("\n[Rack::Lineprof] #{'=' * 63}") + "\n\n" +
+      logger.info Term::ANSIColor.blue("\n[Rack::Lineprof] #{'=' * 63}") + "\n\n" +
```
Comment (kainosnoema) [inline lib/rack/lineprof.rb]: Maybe this should be `debug` instead of `info`?

**Comment by kainosnoema** [comment]
Thanks, I really like this! Just mentioned one thing that might make sense to change, let me know what you think.

**Comment by chiastolite** [comment]
That's for sure! I fix it :smile_cat:

**Comment by kainosnoema** [comment]
Great, thanks! Good addition. :smile:

**Comment by chiastolite** [comment]
Thanks! :tada:

**Pull request merged**